cmd/system-probe/modules/process.go

```diff
@@ -49,11 +51,22 @@ func (t *process) GetStats() map[string]interface{} {
 func (t *process) Register(httpMux *mux.Router) error {
 	var runCounter uint64
 	httpMux.HandleFunc("/proc/stats", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPost {
+			w.WriteHeader(http.StatusMethodNotAllowed)
+			return
+		}
+
 		start := time.Now()
-		stats, err := t.probe.StatsWithPermByPID()
+		pids, err := getPids(req)
 		if err != nil {
-			log.Errorf("unable to retrieve stats using process_tracer: %s", err)
-			w.WriteHeader(500)
+			log.Errorf("%s", err)
```

Comment:
Nit, you could use `log.Error()` instead of `log.Errorf`.

Reply:
Fixed to add an actual message and not just the error.